### ISSUE TITLE
Terraspace fmt don't work on modules

### ISSUE THREAD

**Issue by ByJacob** [boltops-tools/terraspace#116]
## Checklist

- [x] Upgrade Terraspace: Are you using the latest version of Terraspace? This allows Terraspace to fix issues fast. There's an Upgrading Guide: https://terraspace.cloud/docs/misc/upgrading/
- [x] Reproducibility: Are you reporting a bug others will be able to reproduce and not asking a question. If you're unsure or want to ask a question, do so on https://community.boltops.com
- [x] Code sample: Have you put together a code sample to reproduce the issue and make it available?  Code samples help speed up fixes dramatically. If it's an easily reproducible issue, then code samples are not needed. If you're unsure, please include a code sample.

## My Environment

| Software         | Version |
| ---------------- | ------- |
| Operating System |   linux      |
| Terraform        |   0.14.6      |
| Terraspace       |  0.6.9       |
| Ruby             |      ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-linux]   |

---

## Expected Behaviour
Code in modules are formated

## Current Behavior

Code in modules are don't formated

## Step-by-step reproduction instructions
1. Download Sample codes
2. Execute `terraspace fmt`
3. Command format stacks, but don't format modules

## Code Sample

https://github.com/ByJacob/terraspace-bug-issues-116

**Comment by tongueroo** [comment]
By default, only `app/stacks` are formatted by `terraspace fmt`. Can format modules also with a flag:

    terraspace fmt -t all

[Terraspace Docs: terraspace fmt](https://terraspace.cloud/reference/terraspace-fmt/)

Also, can use get the help from the CLI

    terraspace fmt -h

**Comment by ByJacob** [comment]
Haven't seen it before. My mistake :( Why isn't `all` the default?

**Comment by ByJacob** [comment]
I checked this command and `all` is not working. I using prepared sample code.

```
Workspace/GitHub/terraspace-bug-issues-116/sample$ terraspace fmt -t all
Formating terraform files

Workspace/GitHub/terraspace-bug-issues-116/sample$ terraspace fmt -t module
Formating terraform files
app/modules/mod1
main.tf

Workspace/GitHub/terraspace-bug-issues-116/sample$ terraspace fmt -t stack
Formating terraform files
app/stacks/test1
main.tf

```

**Comment by tongueroo** [comment]
Sure. There was a bug in all. It was fixed in #117 as reference above ⬆️ Thanks!
